seed/static/seed/partials/ubid_modal.html

```diff
@@ -23,7 +23,7 @@ <h4 class="modal-title" id="exportModalLabel" ng-show="decode_ubid_state === 're
   <br>
 
   <!-- The summary of selected records is shown in any state. -->
-  <div class="alert alert-info" ng-if="pre_decode_ubid_not_decoded || pre_decode_ubid_not_decoded">
+  <div class="alert alert-info" ng-if="pre_decode_ubid_not_decoded || pre_decode_ubid_not_decoded"><!--TODO: pre_decode_ulid_not_decoded?-->
```

Comment:
Need confirmation.

Reply:
What's the question being asked here?

Reply:
Just noticed that pre_decode_ubid_not_decoded is used twice here? Should one of them be ulid instead?